content/300-guides/200-deployment/110-deployment-guides/600-deploying-migrations-from-a-local-environment.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: 'Deploying migrations from a local environment'
+metaTitle: 'Deploying migrations from a local environment'
+metaDescription: 'Learn how to deploy Node.js and TypeScript applications that are using Prisma Client locally.'
+tocDepth: 2
+---
+
+<TopBlock>
+
+There are instances whereby you might want to run a deployment migration locally. This page outlines some examples of how you can do that and **why its generally a bad practice**.
+
+</TopBlock>
+
+## Local CI/CD pipeline
```

Comment:
Similar feedback here - I think we can really shorten this to:

* If you do not have an automated CI / CD process, you can technically deploy new migrations from your local environment to test and production environments in the following way:

    1. Make sure migration history is up to date (run prisma migrate dev to generate migration history from latest changes)
    1. Change connection string to the the environment (show example)
    1. Run migrate deploy

* Strongly discourage this practice because XYZ (I think we should move this out of the note, as the style is a bit soft / muted). You can use an emoji maybe? "We **strong discourage** ⛔"

* These two say similar things (+ repeats the points you make more strongly in another place about dangers of local deploy) - can we combine into a single short paragraph that says "We recommend setting up an automated CI / CD pipeline to handle deployment to staging and production which uses `migrate deploy` in a pipeline step - see deployment guides for examples`

	  > **Note**: `prisma migrate deploy` should generally be part of an automated CI/CD pipeline, and we do not recommend running this command locally to deploy changes to a production database. It is not generally considered good practice to store the production database URL locally.
	  
	  > An alternative approach would be to let an automated CI/CD pipeline handle your staging and production environments.

* You might not need sub-headings (`###`) after shortening the section, but if you keep them, I suggest renaming to something searchable e.g. "Option 1: Deploy from local" and "Option 2: Automated CI / CD"